pkg/bigint.go

```diff
@@ -0,0 +1,257 @@
+package types
```

Comment:
I would be in favor of moving `bigint`, `fil`, and other bits we consider to be "types" into a directory named `types` -- so `pkg/types/bigint.go` for example.

Reply:
ok i will do this

Reply:
ok i have done this